html/elements/source.json

```diff
@@ -288,7 +288,7 @@
                 "version_added": null
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "9.2"
```

Comment:
ditto

Reply:
Thanks. Looks good. I didn't quite know the correct format for everything.